apps/prairielearn/src/pages/instructorCourseAdminSharing/instructorCourseAdminSharing.ts

```diff
@@ -0,0 +1,94 @@
+import { Router } from 'express';
+import asyncHandler = require('express-async-handler');
+import path = require('path');
+import debugModule from 'debug';
+import error = require('@prairielearn/error');
+import { v4 as uuidv4 } from 'uuid';
+import { InstructorSharing } from './instructorCourseAdminSharing.html';
+import sqldb = require('@prairielearn/postgres');
+
+const router = Router();
+const debug = debugModule('prairielearn:' + path.basename(__filename, '.js'));
+const sql = sqldb.loadSqlEquiv(__filename);
+
+async function generateSharingId(req, res) {
+  const newSharingId = uuidv4();
+  return await sqldb.queryZeroOrOneRowAsync(sql.update_sharing_id, {
+    sharing_id: newSharingId,
+    course_id: res.locals.course.id,
+  });
+}
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    debug('GET /');
+
+    let result = await sqldb.queryOneRowAsync(sql.get_course_sharing_info, {
```

Comment:
Let's use a meaningful name like `sharingInfo` instead of `result`. Let's also use the query functions that validate the results with `zod`. See here for an example:

https://github.com/PrairieLearn/PrairieLearn/blob/6202903e080c69921a97e36b5fa9e726be7cc701/apps/prairielearn/src/ee/pages/institutionAdminCourse/institutionAdminCourse.ts#L23-L30

Reply:
done.